test/common/buffer/owned_impl_test.cc

```diff
@@ -1139,13 +1197,7 @@ TEST_F(OwnedImplTest, ReserveZeroCommit) {
   buf.addBufferFragment(frag);
   buf.prepend("bbbbb");
   buf.add("");
-  constexpr uint32_t reserve_slices = 16;
-  Buffer::RawSlice slices[reserve_slices];
-  const uint32_t allocated_slices = buf.reserve(1280, slices, reserve_slices);
-  for (uint32_t i = 0; i < allocated_slices; ++i) {
-    slices[i].len_ = 0;
-  }
-  buf.commit(slices, allocated_slices);
+  { auto reservation = buf.reserveSingleSlice(1280); }
```

Comment:
reservation.commit(0); may be interesting, although I know is a no-op.

Reply:
Given that this is a regression test, I don't want to change it more than I have to. But I added a `commit(0)` test earlier (line 916 of this file).